### DISCUSSION TITLE
How to write comments?

### DISCUSSION THREAD

**Discussion by borekb** [facebook/docusaurus#3557] [Q&A] [answered]
HTML comments are not supported by MDX, this will produce an error:

```md
<!-- an HTML comment -->
```

MDX comments also don't seem to work for me:

```mdx
{/* this is a comment */}
```

It renders like this:

<img width="240" alt="Screen Shot 2020-10-08 at 12 49 17-shadow" src="https://user-images.githubusercontent.com/101152/95449226-c0be2b00-0964-11eb-87fd-ea965c6c4180.png">

How do I add comments to `.md` / `.mdx` files in Docusaurus?

**Comment by slorber** [comment, marked as answer]
Hi,

Afaik it will be in MDX 2.0.

I don't know if it's provided at MDX  level, Docusaurus level or site level, but on Docusaurus 2 website I know there are comments using this syntax:


```
# Markdown page

<!--

TODO.

-->
```

**Reply by borekb** [reply]
Actually, HTML comments work, I was doing a mistake of not leaving a blank line between the block I wanted to comment and the comment itself. I.e., this works:

```md
<!-- Comment for the paragraph below -->

Paragraph.
```

This doesn't work:

```md
<!-- Comment for the paragraph below -->
Paragraph.
```

So it was my incorrect usage. Thanks for a quick reply!
